pep-9999.rst

```diff
@@ -47,13 +47,57 @@ required to interpret all standard constructs, and additionally a type checker
 can support extra constructs not described in this PEP. All deviations from
 this PEP should be documented.
 
+Syntax and Directory Structure
+==============================
+
+Type stubs are syntactically valid Python files with a ``.pyi`` suffix.
+Type checkers are expected to support the syntax of the latest Python
+version. Type stubs should be parseable with the latest released Python
+version, but also support the previous version up until six months after
+the release of the current version. Older Python versions do not need to be
+supported, even if the implementation supports older versions or the
```

Comment:
I had to read "the implementation supports older versions" a couple times before understanding it - I think it would be clearer to say something along the lines of "the type information in the stub is targeted to an older version".